<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>921</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="VersionPage">
     <widget class="QLabel" name="version_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>490</y>
        <width>154</width>
        <height>42</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="version_number">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>490</y>
        <width>154</width>
        <height>42</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="InfoPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>10</y>
        <width>239</width>
        <height>175</height>
       </rect>
      </property>
      <property name="text">
       <string>My IP Adress</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>180</y>
        <width>486</width>
        <height>269</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>15</y>
        <width>729</width>
        <height>539</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HomePage">
     <widget class="QPushButton" name="ifconfig_button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>IFCONFIG/IP</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>391</width>
        <height>291</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>370</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>361</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="topology_button">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualize</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearpaint_button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Network</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>441</width>
        <height>211</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>560</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionNew"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHome"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string> Version</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
